gnovm/stdlibs/internal/std/frame.gno

```diff
@@ -0,0 +1,18 @@
+package std
+
+type Realm struct {
```

Comment:
can you rebase with master so the diff only show the new things, please?

Reply:
Yes of course
But before that, could you check one issue about PrevRealm? #896 

tbruyelle and I'm having different result